src/helpers/eventEmitter.js

```diff
@@ -0,0 +1,10 @@
+const EventEmitter = require('events');
+
+const eventEmitter = new EventEmitter();
+
+// Registering a listener for the 'passwordUpdated' event
+eventEmitter.on('passwordUpdated', (userId) => {
+  console.log(`Password has been updated for user with ID ${userId}`);
```

Comment:
remove console.log,it is for debugging purpose

Reply:
done